app_dart/README.md

```diff
@@ -4,19 +4,24 @@ This folder contains a Dart based backend for Cocoon.
 
 ## Building and running
 
-#### Prerequisites
+### Prerequisites
 
-You'll need to install the Google Cloud Developer Tools Command Line Interface
-(`gcloud`) in order to do several of the tasks below. Visit
-https://cloud.google.com/sdk/docs/quickstarts for instructions on how to install
-these tools for your platform.
-
-Upon downloading and installing the Google Cloud SDK, you should initialize it
+* Install the Google Cloud Developer Tools Command Line Interface
+([`gcloud`](https://cloud.google.com/sdk/docs/quickstarts)). Then initialize it
 and authenticate yourself by running:
 
 ```sh
+gcloud auth login
 gcloud init
 ```
+* Install Flutter
+```sh
+export PATH="$PATH":"path/to/flutter/bin/"
+flutter upgrade
+flutter pub get
+export PATH="$PATH":"path/to/flutter/bin/cache/dart-sdk/bin/"
+flutter config --enable-web
```

Comment:
This should no longer be needed as it's moved into stable

Reply:
Remoded.